### PR TITLE
GROK-15061: Help : Diff Studio

### PR DESCRIPTION
**Pull request by vmakarichev**
Docs on Diff Studio are updated with respect to its new version release

### CODE REVIEW

File: help/compute/diff-studio.md [hunk 1]
```diff
@@ -171,18 +168,32 @@ You can set new values for parameters and change values for functions.
 
 ![Multi-stage model - loop](pics/DiffStudio-loop.gif)
 
-Apply the `#update` feature to obtain acyclic simulation.
-Set the length of the new modeling stage in the mandatory `length` variable.
-Inside you can use any mathematical expression to modify functions and parameters,
-same as for the `#loop` section.
+### Multi-stage model
```
Comment (skalkin) [inline help/compute/diff-studio.md:171]: Multistage is one word.

File: help/compute/diff-studio.md [hunk 2]
```diff
@@ -171,18 +168,32 @@ You can set new values for parameters and change values for functions.
...
+
+The multi-stage model is a model where parameters and function values can vary for different
```
Comment (skalkin) [inline help/compute/diff-studio.md:173]: Simplify this.

Comment (skalkin) [reply help/compute/diff-studio.md:173]: Something like that:

In multistage model, parameters might change between different argument ranges (stages).

File: help/compute/diff-studio.md [hunk 3]
```diff
@@ -171,18 +168,32 @@ You can set new values for parameters and change values for functions.
...
-  length = 6
-  y = 1
+#argument: t, 1-st stage
+  t0 = 0.01
+  t1 = 15
+  h = 0.01
 ```
 
-Use any valid expression, when specifying the updates of inputs.
+Add the `#update` block. Input name of the stage right after the colon. Set the duration of the new modeling stage in the next line using
```
Comment (skalkin) [inline help/compute/diff-studio.md:185]: > Input name of the stage right after the colon.

Simplify this. And it's "Enter", not "input". And it should be even simpler anyway :)

**Review by skalkin — APPROVED** [review]
Looks good to me, but address my minor comments first. And also I would simplify the block about "update:". OK to merge after that.

File: help/compute/diff-studio.md [hunk 4]
```diff
@@ -145,12 +145,9 @@ Set [tolerance](https://pythonnumericalmethods.berkeley.edu/notebooks/chapter19.
...
 ```
 
-### Multi-stage model
-
-The multi-stage model is a model where parameters and function values can vary for different
-ranges of argument values.
+### Cyclic process simulation
 
-Datagrok provides a special capabilities for multi-stage simulation.
+Datagrok provides a special capabilities for modeling cyclic processes and phenomena.
```
[the remark this answers is not in the document]
Comment (vmakarichev) [reply help/compute/diff-studio.md:150]: Done

File: help/compute/diff-studio.md (hunk 1 above)
Comment (vmakarichev) [reply help/compute/diff-studio.md:171]: Done

File: help/compute/diff-studio.md (hunk 3 above)
Comment (vmakarichev) [reply help/compute/diff-studio.md:185]: I've reduced it

**Pull request merged**